tests/scripts/ci_perf.py

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env python3
+
+# This script is a simulation of what perf.groovy does in CI. It is not currently used for any official purpose, but I wanted
+# to check it in hopes it might be useful to others.
+
+import ctypes
+import json
+import argparse
+import os
+import shutil
+import subprocess
+import sys
+import glob
+from sys import version_info
+
+##########################################################################
+# Argument Parser
+##########################################################################
+
+description = 'Tool to simulate CI perf jobs'
+
+parser = argparse.ArgumentParser(description=description)
+
+#job arguments
+parser.add_argument('-job', dest='job', default=None, required=True, choices=['CoreCLR-Scenarios'])
+parser.add_argument('-isPR', dest='isPR', default=True, choices=[True, False])
+parser.add_argument('-arch', dest='arch', default='x64', choices=['x64', 'x86'])
+parser.add_argument('-os', dest='operatingSystem', default='Windows_NT', choices=['Windows_NT', 'Ubuntu16.04', 'Ubuntu14.04', 'OSX'])
+parser.add_argument('-optLevel', dest='optLevel', default='full_opt', choices=['full_opt', 'min_opt', 'tiered'])
+parser.add_argument('-jitName', dest='jitName', default='ryujit', choices=['ryujit'])
+parser.add_argument('-benchviewCommitName', dest='benchviewCommitName', default='FAKE_BV_COMMIT_NAME')
+parser.add_argument('-bvUploadSasToken', dest='bvUploadSasToken', default='FAKE_BV_SAS_UPLOAD_TOKEN')
```

Comment:
You do not need this. I think it is confusing too.

Reply:
This variable is tight to the `-uploadToBenchview` option. I would check if the environment variable exist when the option below is passed.

Reply:
Also, since you likely won't have the SAS token, you won't be uploading to benchview anyway, so I think the uploadToBenchview option could be eliminated altogether.

Reply:
I'm not sure what environment variable you are referring to? I can remove the option.

Reply:
Then, I think we should remove the `-bvUploadSasToken` and `-uploadToBenchview` options as you will not be uploading to BenchView.